slasher/cache/highest_attestation_cache.go

```diff
@@ -0,0 +1,73 @@
+package cache
+
+import (
+	lru "github.com/hashicorp/golang-lru"
+	slashpb "github.com/prysmaticlabs/prysm/proto/slashing"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	// highestAttCacheSize defines the max number of sets of highest attestation in cache.
+	highestAttCacheSize = 3000
```

Comment:
```suggestion
	highestAttCacheSize = 300000
```

Reply:
no need for so much, the attestations are saved in 1k buckets